[tempo-distributed] Declare after installation

xogoodnow opened the pull request:
Added other components.



p.s: 
Enterprise gateway and the Memcached exporter are not included. should they be included as well?

zalegrala commented:
Including the enterprise components if enterprise is enabled would be a great update too.  Thanks for the PR.

xogoodnow commented:
> Including the enterprise components if enterprise is enabled would be a great update too. Thanks for the PR.

Added all components.
Also changed "gateway.gem.example.com" to "gateway.get.example.com" since its Enterprise Gateway for Traces and not metrics (sorry I'm a little OCD :) ).

Cheers

zalegrala commented:
Alignment in the names with the mimir chart is okay with me.  Do we know if this will be a breaking change?

xogoodnow commented:
> Alignment in the names with the mimir chart is okay with me. Do we know if this will be a breaking change?

Great, I do not think it would be a breaking change.
User can provide a domain for all services which have ingress and then override them when needed and the sub-domain for the service would contain the release name such as below:
query-front.{{ {{ .Release.Name }}.example.com

I would open a PR for this then.
Cheers

xogoodnow commented:
Hi @zalegrala,

Is there anything that I should do for this PR?

xogoodnow commented:
Hi @zalegrala, @mapno ,
What seems to be the issue for this PR to move forward?

xogoodnow commented:
> @xogoodnow, Thanks! for the PR, could you please address the failed CI check.

Hi @Sheikh-Abubaker,
Sure, bumped the version
